packages/domain-search/src/components/DomainSearch/style.scss

```diff
@@ -0,0 +1,5 @@
+@import '@wordpress/base-styles/colors';
+
+.domain-search {
+	--domain-search-secondary-action-color: $gray-900;
```

Comment:
Is this the right way to apply tree-wide overrides to components? I'm setting the variable at the root and then using it down the tree, like [here](https://github.com/Automattic/wp-calypso/pull/104537/files#diff-2a2c76923c50290aed8fd79d11ef3dd2f997b2a8b3a7ca14560250bf85facc5fR75-R83).